org.jacoco.build/pom.xml

```diff
@@ -734,14 +734,6 @@
       </build>
     </profile>
 
-    <!--
-    Following profiles enable compilation into bytecode version 9
-    when requested "bytecode.version" greater than 9,
-    because this is maximum that can be processed by
-    maven-shade-plugin (https://issues.apache.org/jira/browse/MSHADE-289)
```

Comment:
[First builds of JDK 13 EA already available for testing](http://jdk.java.net/13/) and I think same trick will be needed to test new bytecode version, but ok we can restore comment and add profile when will be needed.